lib/active_fedora/fedora_attributes.rb

```diff
@@ -80,10 +89,14 @@ def resource_class
         end
       end
 
-      # Set the rdf type for this class
-      def type(uri)
-        @type = uri
-      end
+      private
+        # @return a Hash of options suitable for passing to ActiveTriples::Base.configure
+        def active_triple_options
+          {}.tap do |h|
```

Comment:
Why is this getting `tap`ped for hash generation? Its a bit confusing.

Why not `{ type: type, rdf_label: rdf_label }`?